services/gitlab/gitlab-merge-requests.service.js

```diff
@@ -0,0 +1,351 @@
+import Joi from 'joi'
+import { optionalUrl, nonNegativeInteger } from '../validators.js'
+import { metric } from '../text-formatters.js'
+import GitLabBase from './gitlab-base.js'
+
+// The total number of MR is in the `x-total` field in the headers.
+// https://docs.gitlab.com/ee/api/index.html#other-pagination-headers
+const schema = Joi.object({
+  'x-total': Joi.number().integer(),
+  'x-page': nonNegativeInteger,
+})
+
+const queryParamSchema = Joi.object({
+  labels: Joi.string(),
+  gitlab_url: optionalUrl,
+}).required()
+
+const documentation = `
+<p>
+  You may use your GitLab Project Id (e.g. 278964) or your Project Path (e.g. gitlab-org/gitlab ).
+  Note that only internet-accessible GitLab instances are supported, for example https://jihulab.com, https://gitlab.gnome.org, or https://gitlab.com/.
+  <a href="https://docs.gitlab.com/ee/user/gitlab_com/index.html#pagination-response-headers">GitLab's API </a> only reports up to 10k Merge Requests, so badges for projects that have more than 10k will not have an exact count.
+</p>
+`
+
+const labelDocumentation = `
+<p>
+  If you want to use multiple labels then please use commas (<code>,</code>) to separate them, e.g. <code>foo,bar</code>.
+</p>
+`
+
+export default class GitlabMergeRequests extends GitLabBase {
+  static category = 'issue-tracking'
+
+  static route = {
+    base: 'gitlab/merge-requests',
+    pattern: ':variant(all|open|closed|locked|merged):raw(-raw)?/:project+',
+    queryParamSchema,
+  }
+
+  static examples = [
+    {
+      title: 'GitLab merge requests',
+      pattern: 'open/:project+',
+      namedParams: {
+        project: 'gitlab-org/gitlab',
+      },
+      queryParams: { gitlab_url: 'https://gitlab.com' },
+      staticPreview: {
+        label: 'merge requests',
+        message: '1.4k open',
+        color: 'brightgreen',
```

Comment:
Sorry to do this again, but I missed this. Can you change all the examples to, preferably, use the `render` function, or at least ensure that the color in the examples is set to blue too?

Reply:
ops...It was an oversight on my part.